addons/l10n_it_edi/models/account_move.py

```diff
@@ -0,0 +1,1212 @@
+        try:
+            return datetime.fromisoformat(datetime_str.replace('Z', '+00:00'))
+        except (ValueError, TypeError):
+            return False
+    return False
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_it_edi_state = fields.Selection([
+        ('being_sent', 'Being Sent To SdI'),
+        ('requires_user_signature', 'Requires user signature'),
+        ('processing', 'SdI Processing'),
+        ('rejected', 'SdI Rejected'),
+        ('forwarded', 'SdI Accepted, Forwarded to Partner'),
+        ('forward_failed', 'SdI Accepted, Forward to Partner Failed'),
+        ('forward_attempt', 'SdI Accepted, Forwarding to Partner'),
+        ('accepted_by_pa_partner', 'SdI Accepted, Accepted by the PA Partner'),
+        ('rejected_by_pa_partner', 'SdI Accepted, Rejected by the PA Partner'),
+        ('accepted_by_pa_partner_after_expiry', 'SdI Accepted, PA Partner Expired Terms'),
+    ], copy=False, tracking=True, readonly=True)
+    l10n_it_edi_header = fields.Html(
+        help='User description of the current state, with hints to make the flow progress',
+        readonly=True,
+        copy=False)
+    l10n_it_edi_transaction = fields.Char(copy=False, string="FatturaPA Transaction")
+    l10n_it_edi_attachment_file = fields.Binary(copy=False, attachment=True)
+    l10n_it_edi_attachment_id = fields.Many2one(
+        comodel_name='ir.attachment',
+        string="FatturaPA Attachment",
+        compute=lambda self: self._compute_linked_attachment_id('l10n_it_edi_attachment_id', 'l10n_it_edi_attachment_file'),
+        depends=['l10n_it_edi_attachment_file']
+    )
+    l10n_it_edi_is_self_invoice = fields.Boolean(compute="_compute_l10n_it_edi_is_self_invoice")
+    l10n_it_stamp_duty = fields.Float(default=0, string="Dati Bollo")
+    l10n_it_ddt_id = fields.Many2one('l10n_it.ddt', string='DDT', copy=False)
+
+    def action_check_l10n_it_edi(self):
+        self.ensure_one()
+        if not self.l10n_it_edi_transaction and self.l10n_it_edi_state not in WAITING_STATES:
+            raise UserError(_("This move is not waiting for updates from the SdI."))
+        self.company_id.account_edi_proxy_client_ids.filtered(
+            lambda x: x.proxy_type == "l10n_it_edi")._l10n_it_edi_update_send_state(self, interactive=True)
+
+    @api.depends('l10n_it_edi_transaction')
+    def _compute_show_reset_to_draft_button(self):
+        super()._compute_show_reset_to_draft_button()
+        for move in self.filtered(lambda m: m.l10n_it_edi_transaction):
+            move.show_reset_to_draft_button = False
+
+    def _get_edi_decoder(self, file_data, new=False):
+        # EXTENDS 'account'
+        if file_data['type'] == 'l10n_it_edi':
+            return self._l10n_it_edi_import
+        return super()._get_edi_decoder(file_data, new=new)
+
+    def _post(self, soft=True):
+        super()._post(soft)
+        for move in self:
+            move.l10n_it_edi_header = False
```

Comment:
```suggestion
        self.write({'l10n_it_edi_header': False})
```
Plz add comment + `EXTENDS 'account'`